test/e2e/handler/utils.go

```diff
@@ -539,3 +539,15 @@ func skipIfNotKubernetes() {
 		Skip("Tutorials use interface naming that is available only on Kubernetes providers")
 	}
 }
+
+func maxUnavailableNodeCount() int {
```

Comment:
As said let's try to use the same logic from porduct moving the function under `pkg` and usint it on tests.